### ISSUE TITLE
Update teidata.temporal.iso (and teidata.duration.iso?) 

### ISSUE THREAD

**Issue by peterstadler** [TEIC/TEI#1885]
This feature request was originally brought up by Thomas Stäcker on [TEI-L](http://tei-l.970651.n3.nabble.com/att-datable-iso-and-ISO-8601-2-2019-tp4031945.html):

> am I right that the extensions to ISO 8601 (ISO 8601-2:2019 - https://www.iso.org/standard/70908.html)  are not yet valid in att.datable.iso? In this ISO extension for dates and time new concepts were introduced namly vagueness expressed by ~, uncertainty  expressend by ? and a combination of vagueness and uncertainty expressed by %, e.g. <date when-iso="1950~">around the year 1950</date>. If not already done I'd recommend to adapt the schema to this new version of  ISO 8601.

**Comment by ebeshero** [comment]
Council F2F agrees we should move in direction of current ISO conformance, but this needs review to see how the ISO release (of February 2019) affects our current practice of encoding ambiguous dates.

**Comment by sydb** [comment]
First, what _are_ our current recommendations for encoding ambiguous dates?

Anyway, I have taken a quick stab here at making a list of things we need to do in order to update to ISO 8601-1:2019 and ISO 8601-2:2019 (which I personally think is a very good idea). Basically, most of the work needs to be done in 13.3.7.3 _More Expressive Normalizations_ (`NDDATEISO`)

- [ ] Expand the list of capabilities in the last half of the 1st paragraph 
- [ ] Add another example to the end of that section that shows something that could not be expressed in ISO 8601:2004, e.g. `when-iso="{1960,1962,1964}"` (each of the 3 years) and `when-iso="[1960-01,1962-10,1964-06]"` (one of the 3 months).
- [ ] Add subsections or examples for the various new capabilities.

Furthermore, the regular expression restriction for att.temporal.iso will need to be updated. A lot.

**Comment by martindholmes** [comment]
Should repeating intervals be included? In other words, should we allow:

R52/2025-01-03T08:00:00Z/P7D

meaning every week for 52 weeks at 8 am starting January 3 2025?

I've seen no requests for this, but it is easy to imagine coming across the phrase "every Friday throughout 2025" and wanting to encode it in a processable manner.

**Comment by sydb** [comment]
My _Dates and Times in DH_ document includes repeating intervals (but neither grouped time scale units nor selection rules). So my instinct is “yes” we should allow repeating intervals out of the gate.

P.S. I got a _lot_ done on it over break. Hope to have it in shape for review within a month.

**Comment by martindholmes** [comment]
@sydb I believe you updated us on this at the conference this week -- can you summarize for the ticket?
